repack_st_arch.csh

```diff
@@ -680,6 +689,7 @@ if ($do_history == true) then
       #            "TYPE_${INST}.eo " \
       set cmds_template = cmds_template_$models[$m]
 
+      echo "Checking existence of the template used to make a local cmds file."
```

Comment:
I understand the template has to live somewhere ... why that location is being reported is confusing. I read the log file
and it says we are working in the lnd/hist directory but the reports stuff about cpl/hist ... and then follows it with 'ls' errors. Scary.